orbital/models.py

```diff
@@ -27,15 +28,15 @@ def notify_orbital(ip, data=None, **kwargs):
         return
 
     if data:
-        platform = (data.get('platform') or 'other').lower()
+        platform = (data.get("platform") or "other").lower()
     else:
-        platform = ''
+        platform = ""
 
     data = [
-        round(result['latitude'], 4),
-        round(result['longitude'], 4),
+        round(result["latitude"], 4),
+        round(result["longitude"], 4),
         int(time() * 1000),
         platform,
     ]
 
-    udp_socket.sendto(dumps(data), udp_addr)
+    udp_socket.sendto(dumps(data).encode("utf-8"), udp_addr)
```

Comment:
Does the `udp_addr` need to be bytes as well?

Reply:
![image](https://user-images.githubusercontent.com/1421724/99320406-698f5e00-2820-11eb-9499-66cc1878c9f4.png)

nope